api/app/Models/Pool.php

```diff
@@ -105,7 +105,48 @@ public function getActivitylogOptions(): LogOptions
 
     public function user(): BelongsTo
     {
-        return $this->belongsTo(User::class);
+        // avoid selecting searchable column from user table
+        // avoid selecting searchable column from user table
+        return $this->belongsTo(User::class)
+            ->select(['id',
+                'email',
+                'first_name',
+                'last_name',
+                'telephone',
+                'preferred_lang',
+                'current_province',
+                'current_city',
+                'looking_for_english',
+                'looking_for_french',
+                'looking_for_bilingual',
+                'bilingual_evaluation',
+                'comprehension_level',
+                'written_level',
+                'verbal_level',
+                'estimated_language_ability',
+                'is_gov_employee',
+                'has_priority_entitlement',
+                'priority_number',
+                'department',
+                'current_classification',
+                'citizenship',
+                'armed_forces_status',
+                'is_woman',
+                'has_disability',
+                'is_visible_minority',
+                'has_diploma',
+                'location_preferences',
+                'location_exemptions',
+                'position_duration',
+                'accepted_operational_requirements',
+                'gov_employee_type',
+                'priority_weight',
+                'indigenous_declaration_signature',
+                'indigenous_communities',
+                'preferred_language_for_interview',
+                'preferred_language_for_exam',
+                'deleted_at',
+            ]);
```

Comment:
Sucks we can't exclude columns 😢 This is quite a large array that looks to be mirrored in `PoolCandidate`. Would it be worth making this a property of the `User` model so as we modify that model we only have to update in one place? Something like `User::relatedColumns`? Not sure about the name, but the concept could be useful.

Reply:
This is a great suggestion.  Fixed.